src/clj/rems/db/applications.clj

```diff
@@ -1018,15 +1022,15 @@
 (defn- valid-user? [userid]
   (not (nil? (users/get-user-attributes userid))))
 
-(defn- validate-form [application-id]
-  (let [user-id (get-applicant-of-application application-id)
-        validation (form-validation/validate (get-form-for user-id application-id))]
-    (when-not (= :valid validation)
-      validation)))
+(defn- get-form [form-id]
+  (-> ((resolve 'rems.db.form/get-form) form-id) ;; XXX: avoid cyclic dependency
```

Comment:
this is bad. do you have a plan for getting rid of it right after this PR?

Reply:
`rems.db.form` depends on `rems.db.applications`. `rems.db.applications/process-item` is applications ns even though it deals mostly with form fields. After removing the round-based workflow this will be easier to fix.

Reply:
The new form API might also fix this.

Reply:
I'd just move proces-item to rems.db.form

Reply:
👌